http/src/main/resources/reference.conf

```diff
@@ -152,6 +152,7 @@ vpc {
   highSecurityProjectSubnetworkLabel = "vpc-subnetwork-name"
   networkName = "leonardo-network"
   networkTag = "leonardo"
+  privateAccessNetworkTag = "leonardo-private"
```

Comment:
where is this tag being used?

Reply:
This tag is set when we create the cluster in `DataprocInterpreter`. It matches a [firewall rule](https://github.com/DataBiosphere/terra-resource-buffer/pull/193/files#diff-dd28bc9005c6828e422edf6d2f9cf50ea66531685d27be1f36fc46478af308c6R190) defined in RBS to block outgoing traffic.

Reply:
ok..is the screenshot still valid?

Reply:
Ah - the screenshot is valid if you `s/leonardo-worker/leonardo-private`